http/src/main/resources/init-resources/rstudio-docker-compose.yaml

```diff
@@ -20,11 +19,9 @@ services:
       RUNTIME_NAME: "${RUNTIME_NAME}"
       OWNER_EMAIL: "${OWNER_EMAIL}"
     volumes:
-      - /work:/home/rstudio
-      - /etc/rstudio-license-file.lic:/etc/rstudio-server/license.lic:ro
+      - ${WORK_DIRECTORY}:/home/rstudio
     env_file:
-      - /etc/google_application_credentials.env
-      - /etc/custom_env_vars.env
+      - /var/custom_env_vars.env
     # See https://docs.docker.com/engine/reference/run/#user-memory-constraints
     mem_limit: ${MEM_LIMIT}   # hard limit on memory consumption by the container
```

Comment:
Need to add
```
networks:
  app_network:
    external: true
```
?

Reply:
I think it's not needed since `docker -f xxx -f xxx ... config` will merge all the config files, and this block is already added in `network.yaml`

Reply:
Gotcha.. also missed that we now have `rstudio-docker-compose-gce.yaml`